src/compilers/project.json

```diff
@@ -8,8 +8,8 @@
       "type": "platform",
       "version": "1.0.0-rc2-23931"
     },
-    "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160329-01",
-    "Microsoft.Net.Compilers.netcore": "1.3.0-beta1-20160329-01",
+    "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160405-05",
```

Comment:
This project is here to generate portable deps files for `csc.exe` and `vbc.exe`. So in theory removing roslyn from here should work